data/templates/default/init.lua

```diff
@@ -780,5 +797,234 @@ return {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after,
+      },
+      ingame_image_line = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after + ingame_padding,
+      },
+      ingame_lore_author = {
+         font = {
+            color = ingame_text_color,
+            face = ingame_font_face,
+            size = 10,
+            italic = true,
+         },
+         align = "right",
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after + 3,
+      },
+      about_text = {
+         font = {
+            color = fs_font_color,
+            face = fs_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before,
+         space_after = ingame_text_space_after,
+      },
+      about_sparse = {
+         font = {
+            color = fs_font_color,
+            face = fs_font_face,
+            size = 12,
+         },
+         space_before = ingame_text_space_before + ingame_padding,
+         space_after = ingame_text_space_after,
+      },
+
+
+      -- Returned when lookup by name fails.
+      -- Make this stand out to help debugging
+      unknown = {
+         font = {
+            color = unknown_font_color,
+            face = unknown_font_face,
+            size = unknown_font_size,
+            italic = true,
+         },
+         space_before = unknown_font_size,
+         space_after = unknown_font_size,
+      },
    },
+
+   colors = {
+      campaign_bar_thron = campaign_leader_color,
+      campaign_bar_boldreth = {255, 0, 0},
+      campaign_bar_khantrukh = {128, 128, 255},
+
+      campaign_emp_lutius = campaign_leader_color,
+      campaign_emp_amalea = {255, 0, 0},
+      campaign_emp_saledus = {143, 145, 49},
+      campaign_emp_marcus = {17, 117, 175},
+      campaign_emp_julia = {208, 208, 208},
+
+      campaign_atl_jundlina = campaign_leader_color,
+      campaign_atl_sidolus = {255, 26, 48},
+      campaign_atl_loftomor = {253, 213, 61},
+      campaign_atl_colionder = {51, 169, 253},
+      campaign_atl_opol = {255, 63, 251},
+      campaign_atl_ostur = {55, 95, 252},
+      campaign_atl_kalitath = {255, 0, 0},
+
+      campaign_fri_reebaud = {85, 187, 85},
+      campaign_fri_hauke = {119, 153, 187},
+      campaign_fri_maukor = {153, 0, 0},
+      campaign_fri_murilius = {187, 119, 85},
+      campaign_fri_claus = {28, 194, 0},
+      campaign_fri_henneke = {247, 255, 0},
+      campaign_fri_iniucundus = {255, 0, 89},
+      campaign_fri_angadthur = {0, 22, 255},
+      campaign_fri_amazon = {108, 0, 151},
+      campaign_fri_ketelsen = {68, 85, 204},
```

Comment:
I am not sure if those campaign specific colors should be in this file. From my understanding this file is used for general UI styles, not for campaign related things. E.g. Custom themes are based on this file.

Reply:
There are several reasons why I think they belong here:

1. there is no other place where the scenarios of a campaign can store common values
2. they may need changing if someone wants to create a theme with a very different colour scheme, especially if it is aimed at visually impaired users
3. these campaigns are official game content

Actually for similar reasons (except for 3. of course, or even that if an addon aims to become official content), it could be useful to come up with a way to allow addons to add their own style elements which theme addons can modify as well in the long run.